packages/main/src/List.js

```diff
@@ -567,21 +566,9 @@ class List extends UI5Element {
 
 	get classes() {
 		return {
-			main: {
-				sapMList: true,
-				sapMListInsetBG: this.inset,
-				sapUiSizeCompact: getCompactSize(),
-			},
-			ul: {
-				sapMListItems: true,
-				sapMListUl: true,
-				[`sapMListShowSeparators${this.separators}`]: true,
-				[`sapMListMode${this.mode}`]: true,
-				sapMListInset: this.inset,
-			},
 			noData: {
+				"ui5-list-nodata": true,
 				sapMLIB: true,
```

Comment:
These classes ("sapMLIB", "sapMLIBTypeInactive") would be changed with ListItem/StandardListItem styles refactoring